Source/FileLiberator/DownloadDecryptBook.cs

```diff
@@ -158,194 +156,40 @@ AaxcDownloadConvertBase converter
-		Using the example above, if "Book 1" was 15 seconds long and "Part 3" was 20 seconds long:
-
-		00:00 - 00:10	Opening Credits
-		00:10 - 00:25	Book 1
-		00:25 - 00:27	|	Part 1
-		00:27 - 01:40	|	|	Chapter 1
-		01:40 - 03:20	|	|	Chapter 2
-		03:20 - 03:22	|	Part 2
-		03:22 - 05:00	|	|	Chapter 3
-		05:00 - 06:40	|	|	Chapter 4
-		06:40 - 06:42	Book 2
-		06:42 - 07:02	|	Part 3
-		07:02 - 08:20	|	|	Chapter 5
-		08:20 - 10:00	|	|	Chapter 6
-		10:00 - 10:02	|	Part 4
-		10:02 - 11:40	|	|	Chapter 7
-		11:40 - 13:20	|	|	Chapter 8
-		13:20 - 13:30	End Credits
-
-		then flattenChapters will combine them into this:
-
-		00:00 - 00:10	Opening Credits
-		00:10 - 00:25	Book 1
-		00:25 - 01:40	Book 1: Part 1: Chapter 1
-		01:40 - 03:20	Book 1: Part 1: Chapter 2
-		03:20 - 05:00	Book 1: Part 2: Chapter 3
-		05:00 - 06:40	Book 1: Part 2: Chapter 4
-		06:40 - 07:02	Book 2: Part 3
-		07:02 - 08:20	Book 2: Part 3: Chapter 5
-		08:20 - 10:00	Book 2: Part 3: Chapter 6
-		10:00 - 11:40	Book 2: Part 4: Chapter 7
-		11:40 - 13:20	Book 2: Part 4: Chapter 8
-		13:20 - 13:40	End Credits
-
-		*/
-
-        public static List<Chapter> flattenChapters(IList<Chapter> chapters, string titleConcat = ": ")
-        {
-            List<Chapter> chaps = new();
-
-            foreach (var c in chapters)
-            {
-                if (c.Chapters is null)
-					chaps.Add(c);
-                else if (titleConcat is null)
-                {
-                    chaps.Add(c);
-					chaps.AddRange(flattenChapters(c.Chapters));
-                }
-				else
-				{
-                    if (c.LengthMs < 10000)
-                    {
-                        c.Chapters[0].StartOffsetMs = c.StartOffsetMs;
-                        c.Chapters[0].StartOffsetSec = c.StartOffsetSec;
-                        c.Chapters[0].LengthMs += c.LengthMs;
-                    }
-                    else
-                        chaps.Add(c);
-
-                    var children = flattenChapters(c.Chapters);
-
-                    foreach (var child in children)
-                        child.Title = $"{c.Title}{titleConcat}{child.Title}";
-
-                    chaps.AddRange(children);
-                }
+                File.WriteAllText(metadataFile, item.SourceJson.ToString());
+                OnFileCreated(libraryBook, metadataFile);
             }
-            return chaps;
-        }
+			return success;
+		}
 
-        public static void combineCredits(IList<Chapter> chapters)
-        {
-            if (chapters.Count > 1 && chapters[0].Title == "Opening Credits")
-            {
-                chapters[1].StartOffsetMs = chapters[0].StartOffsetMs;
-                chapters[1].StartOffsetSec = chapters[0].StartOffsetSec;
-                chapters[1].LengthMs += chapters[0].LengthMs;
-                chapters.RemoveAt(0);
-            }
-            if (chapters.Count > 1 && chapters[^1].Title == "End Credits")
+		private void Converter_RetrievedMetadata(object sender, AAXClean.AppleTags tags)
```

Comment:
Unlike the AAX(C) files, Mpeg-DASH wiles delivered by widevine contain no metadata.